src/content/doc-sdk-golang/data-types.mdx

```diff
@@ -151,6 +191,9 @@ Where surrealdb.O is defined below. There is no special advantage in using this
 type surrealdb.O map[interface{}]interface{}
 ```
 
+> [!NOTE]
+> `NewGeometryPoint` takes the latitude and the longtitude in an order that conflicts with SurrealDB. Based on GeoJSON, SurrealDB assumes the longtitude appear earlier, while `NewGeometryPoint` takes the latitude earlier. TThe function should be changed to take the longtitude earlier. Related issue: #223
```

Comment:
Can you use `models.GeometryPoint` directly in the above examples so that there's no confusion and we can safely deprecate and delete the NewGeometryPoint?

```golang
GeometryPoint{
  Longitude: 22.00,
  Latitude: -0.11,
}
```